tutorials/experiments/langchain_email_extraction.ipynb

```diff
@@ -1,269 +1,2445 @@
 {
```

Comment:
<p>Can we add some interpretation to the results here? How has the metric moved, and what does that mean in terms of the performance of the application with the different models?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/3805/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>